src/index.html

```diff
@@ -6,22 +6,328 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Catalog</title>
+    <title>Product cards</title>
+    <link
+      rel="stylesheet"
+      href="./styles/index.scss"
```

Comment:
In the <link> tag, the 'href' attribute should point to a CSS file, not an SCSS file. SCSS files are preprocessed into CSS files which browsers can understand. Make sure you point to the correct CSS file generated from your SCSS.